simulator/mips/mips_instr.cpp

```diff
@@ -139,6 +139,15 @@ template<typename I> auto mips_xor     = ALU::xorv<I>;
 template<typename I> auto mips_xori    = ALU::xori<I>;
 template<typename I> auto mips_unknown = ALU::unknown_instruction<I>;
 
+//CP1 instructions
+template<typename I> auto mips_abs_s   = ALU::unknown_instruction<I>;
```

Comment:
Please use unknown_instruction everywhere, as we do not guarantee correctness of execution yet.